src/components/Avatar/Avatar.tsx

```diff
@@ -19,7 +19,8 @@ interface IAvatarState {
 @observer
 export class Avatar extends React.Component<IAvatarProps, IAvatarState> {
   static defaultProps = {
-    source: '/images/default-image-square.png'
+    source: '/images/default-image-square.png',
+    big: false
```

Comment:
Is it possible to do here something like this:
here use `avataClass: ''` with empty string, 
than in the header `<Avatar avatarClass="avatar--big" />` and just <Awatar /> in other places.

in this JSX in all places: 
```<div className='avatar ${this.props.avatarClass}'>```